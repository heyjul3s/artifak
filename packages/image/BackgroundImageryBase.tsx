import { StyledComponent } from 'styled-components';
import { createStyledComponent } from '@artifak/component-generator';
import { Base } from './typings';

export const BackgroundImageryBase: StyledComponent<
  'div',
  any,
  Base.BackgroundImagery,
  any
> = createStyledComponent<Base.BackgroundImagery>();
