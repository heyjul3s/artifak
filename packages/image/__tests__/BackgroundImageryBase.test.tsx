import React from 'react';
import { render } from '@testing-library/react';
import { BackgroundImageryBase } from '../BackgroundImageryBase';

describe('BackgroundImageryBase', () => {
  it('should render', () => {
    const { asFragment } = render(<BackgroundImageryBase />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
