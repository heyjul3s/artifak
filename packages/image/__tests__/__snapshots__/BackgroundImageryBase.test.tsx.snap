// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundImageryBase should render 1`] = `
<DocumentFragment>
  <div
    class="artifak__sc-6om8fa-0 dtstAK"
  />
</DocumentFragment>
`;
